flink-python/pom.xml

```diff
@@ -298,7 +298,7 @@ under the License.
 			<!-- Indirectly accessed in pyflink_gateway_server -->
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-sql-connector-rabbitmq</artifactId>
-			<version>${project.version}</version>
+			<version>3.0.0-1.16</version>
```

Comment:
That's related to [my comment in the jdbc removal PR](https://github.com/apache/flink/pull/21536#discussion_r1056276909)